packages/pds/src/context.ts

```diff
@@ -256,12 +263,36 @@ export class AppContext {
       appviewCdnUrlPattern: cfg.bskyAppView?.cdnUrlPattern,
     })
 
+    const safeAgent = new undici.Agent({
```

Comment:
I almost want to call this `proxyAgent` or `safeProxyAgent` or something similar to give the sense of what it's for since it's only use there